cmd/gateway-gcs.go

```diff
@@ -17,23 +17,21 @@
 package cmd
 
 import (
-	"bytes"
+	"cloud.google.com/go/storage"
 	"context"
 	"crypto/sha256"
 	"encoding/base64"
 	"encoding/hex"
 	"fmt"
+	"google.golang.org/api/googleapi"
+	"google.golang.org/api/iterator"
 	"hash"
 	"io"
 	"path"
 	"regexp"
 	"strconv"
 	"strings"
 
-	"cloud.google.com/go/storage"
```

Comment:
We should avoid imports being organized in between other standard libraries.. https://github.com/golang/go/wiki/CodeReviewComments#imports

```Imports are organized in groups, with blank lines between them. The standard library packages are always in the first group.```